src/assets/Directory.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getDirectory, follow, unfollow, fetchUser } from '../store/actions';
+import { Layout, List, Card, Avatar, Button } from 'antd';
+
+const Directory = (props) => {
+  const { Header, Content } = Layout;
+
+  useEffect(() => {
+    props.fetchUser();
+    props.getDirectory();
+  }, [props.user.following]);
+
+  const isFollowing = (user_id) => {
+    return props.user.following_list.find(
+      (item) => item.following_id === user_id
+    );
+  };
+
+  return (
+    <div>
+      <Layout style={{ minHeight: '100vh' }}>
+        <Header
+          style={{
+            padding: '0px 0px',
+            background: '#f0f2f5',
+            display: 'flex',
+            justifyContent: 'flex-start',
+            height: 'auto',
+          }}
+        >
+          <h2>Directory</h2>
+        </Header>
+        <Content>
+          <List
+            grid={{ gutter: 16, column: 2 }}
+            dataSource={props.users}
+            pagination={{
+              onChange: (page) => {},
+              pageSize: 6,
+            }}
+            renderItem={(user) => (
+              //   props.user.id !== user.id && (
```

Comment:
We can get rid of this since you moved it to line 58.

Reply:
I'll probably make this change before you when I go into the file to move over the styles

Reply:
done

Reply:
Thank you